frontend/src/app/patients/PatientForm.jsx

```diff
@@ -130,6 +130,12 @@ const Fieldset = (props) => (
   </fieldset>
 );
 
+// TODO: move to an Address component
```

Comment:
We have several address entry groups in the app, longer term we should make them look and work consistently with an Address component.